codecrumbs.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
1) please move this file to example-project dir (and fill with real values, not nulls)
2) please add info how to use to Readme
3) please check format for keys (e.g. there is entryPoint but no entryFile, ideally to refactor here a bit to have one format for keys)